### PR TITLE
version one of challege one

### PR DESCRIPTION
**Pull request by susanreenesaa**
@dennisja Please review my work
Input /output / variables/ conditional logic:
Write a javascript program that prompts a user to enter the user's year of birth and
in turn prints a string in the console stating whether the user is a minor, a youth or an elder. Anyone below 18 years is a minor, anyone between 18 and 36 years of
age is a youth and the rest are elders.

### CODE REVIEW

**Comment by dennisja** [comment]
Put the question you were trying to attempt in the description of the pull request. I would like to know what you were trying to achieve and see how we can make it better

**Comment by dennisja** [comment]
Just Indicate the description of what you have done in the description. Don't include anything this pull request doesn't implement

File: ChallengeOneWeek2.html [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
-  <body></body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head> </head>
+
+  <body>
+    <script>
+      while (true) {
+        var birthyear = prompt("enter year of birth");
```
Comment (dennisja) [inline ChallengeOneWeek2.html]: Use `camelCase` when naming your variables i.e `birthYear` instead of `birthyear`.

**TLDR;**
Usually, language communities have come up with conventions of naming variables. In the JavaScript community and codebases, `camelCase` is commonly used for naming variables and functions, whereas `PascalCase` is commonly used for naming classes. This is usually done for readability because code is more read than written. The code you write today will be read by someone else on your team tomorrow. For example `aVeryLongVariableName` is easy to read compared to `averylongvariablename`.

**Review by dennisja — CHANGES_REQUESTED** [review]

File: ChallengeOneWeek2.html (hunk 1 above)
Comment (susanreenesaa) [reply ChallengeOneWeek2.html]: added camelCase. thanks for the correction
